mu/modes/circuitpython.py

```diff
@@ -64,6 +64,8 @@ class CircuitPythonMode(MicroPythonMode):
         (0x04D8, 0xEAD2, None, "DynOSSAT-EDU-OBC"),
         (0x1209, 0x4DDD, None, "ODT CP Sapling M0"),
         (0x1209, 0x4DDE, None, "ODT CP Sapling M0 w/ SPI Flash"),
+        (0x239A, 0x80AC, None, "Unexpected Maker FeatherS2"),
+        (0x303A, 0x8002, None, "Unexpected Maker TinyS2"),
```

Comment:
Was this allocated via this repo? https://github.com/espressif/usb-pids

Reply:
Yes, TinyS2 was issued from Espressif as part of their new allocating PIDs to their customers